src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java

```diff
@@ -284,6 +305,58 @@ public void actionPerformed(ActionEvent e) {
       WEST, HORIZONTAL, insets, 0, 0));
   }
 
+  private void exportConfiguration() {
+    if (project == null) return;
+
+    runOnUiThread(project, () -> {
+      if (confirm(project)) {
+        System.out.println("Confirmed to share");
+      }
+    });
+  }
+
+  private static boolean confirm(Project project) {
+    var binding = SonarLintUtils.getService(project, ProjectBindingManager.class).getBinding();
+    var isSonarCloud = false;
+
+    if(binding == null) {
+      GlobalLogOutput.get().log("Binding is not available", ClientLogOutput.Level.ERROR);
+      return false;
+    }
+
+    try {
+      isSonarCloud = SonarLintUtils.getService(project, ProjectBindingManager.class).getServerConnection().isSonarCloud();
+    } catch (InvalidBindingException e) {
+      GlobalLogOutput.get().log(e.getMessage(), ClientLogOutput.Level.ERROR);
+      return false;
+    }
+
+    var projectKey = binding.getProjectKey();
+    var url = binding.getConnectionName();
+
+    String orgString;
```

Comment:
`orgString` is a bit strange as the text can be about SonarQube, it could be something like `connectionKind` instead